module.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Psr\Container\ContainerInterface;
+use RebelCode\Bookings\WordPress\Module\UIModule;
+
+return function (ContainerInterface $c) {
+    return new UIModule($c->get('container-factory'));

```

Comment:
As per our recent agreement, here and wherever else necessary, change the service keys declared to `snake_case`. Also assume that services provided by other modules are also in `snake_case`.